api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -334,7 +344,22 @@ private void validateStringLength(String field, String fieldName, int max, int m
     }
   }
 
-  public List<Profile> listAllProfiles() {
-    return userService.getAllUsers().stream().map(this::getProfile).collect(Collectors.toList());
+  public void adminUpdateProfile(Profile updatedProfile) {
+    final DbUser user = profileMapper.profileToDbUser(updatedProfile);
```

Comment:
OK, so you're making a new user every time? That doesn't sound like we are using the ORM system right. Can't we make edits on the existing one and save that?

If you ran into some issue, please comment on that before trying to manually fixup the profile. That seems really error prone and difficult to maintain.